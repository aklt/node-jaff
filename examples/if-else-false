before
#if false
branch 1
#else
branch 2
#end
after
