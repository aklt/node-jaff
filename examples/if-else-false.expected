before
branch 2
after
